manim/mobject/geometry/labeled.py

```diff
@@ -0,0 +1,154 @@
+r"""Mobjects that inherit from lines and contain a label along the length."""
+
+from __future__ import annotations
+
+__all__ = ["LabeledLine", "LabeledArrow"]
+
+from manim.constants import *
```

Comment:
This is currently unused and could be removed... but maybe this is imported because the label's *would* use the constant, `DEFAULT_FONT_SIZE`? This brings up the question of what is a sensible default for font_size... perhaps 15 is too small?

Reply:
Thank you. It could be set at a higher value, although in some cases, I have used smaller as well. For example, the short video at the top of this PR is using a font size of 10. What value do you think is more appropriate as default?

Reply:
I added the `DEFAULT_FONT_SIZE` as per the suggestion. The small value was probably just specific to my use case. Thanks for the suggestion.